x-pack/plugins/observability_solution/observability_ai_assistant/common/conversation_complete.ts

```diff
@@ -104,7 +104,8 @@ export type StreamingChatResponseEvent =
   | ConversationUpdateEvent
   | MessageAddEvent
   | ChatCompletionErrorEvent
-  | TokenCountEvent;
+  | TokenCountEvent
+  | BufferFlushEvent;
```

Comment:
`BufferFlushEvent` c[an be returned as a streaming chat response event in cloud](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_ai_assistant/server/routes/chat/route.ts#L166)

Reply:
I only added it due to causing a type error in [complete.spec.ts](https://github.com/elastic/kibana/pull/192219#discussion_r1750613004).  Is it okay to leave it?